romancal/jump/jump_step.py

```diff
@@ -25,9 +25,9 @@ class JumpStep(RomanStep):
     """
 
     spec = """
-        rejection_threshold = float(default=4.0,min=0) # CR sigma rej thresh
-        three_group_rejection_threshold = float(default=6.0,min=0) # CR sigma rej thresh
-        four_group_rejection_threshold = float(default=5.0,min=0) # CR sigma rej thresh
+        rejection_threshold = float(default=180.0,min=0) # CR sigma rej thresh
+        three_group_rejection_threshold = float(default=185.0,min=0) # CR sigma rej thresh
+        four_group_rejection_threshold = float(default=190.0,min=0) # CR sigma rej thresh
```

Comment:
@ddavis-stsci Did you run any tests when choosing these values? Just curious as to how you determined them.

Reply:
I just used the values from JWST as a start and just ran trial and error till things "looked" right. Nothing deep.